### PR TITLE
1130 Updating and simplifying Frida detection section

### PR DESCRIPTION
**Pull request by cpholguera**
This PR covers issue #1130.

I took the chance to solve the issue and additionally compact the section.

- Removed code that can be anyway read in Bernd's article (to avoid maintaining code/detections that might outdate in the future).
- Removed checks being too trivial to bypass.
- Added a clear disclaimer on the effectiveness of these detections.
- Added a clear overview over some of the methods including a brief description and discussion.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 8e8fe3f7e2c62bd5740f57b8b957994085e289fd: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md [hunk 1]
```diff
@@ -809,81 +809,36 @@ Reverse engineers use a lot of tools, frameworks, and apps, many of which you've
...
-
-    return returnValue;
-}
-
-```
-
-Starting with Android 7.0 (API level 24) the `ps` command will only return processes started by the user itself, which is due to a stricter enforcement of namespace separation to increase the strength of the [Application Sandbox](https://source.android.com/security/app-sandbox "Application Sandbox") . When executing `ps` it will read the information from `/proc` and it's not possible to access information that belongs to other user ids.
-
-![Executing ps on Android 5.0 (API level 21)](Images/Chapters/0x05j/Android_Lollipop_ps.png)
+**Method**: A working solution to detect the frida-server process is to inspect the running services and processes. However,
```
Comment (sushi2k) [inline Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md]: "However," should be removed

Comment (cpholguera) [reply Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md]: I like the idea of the table ;)

**Comment by OMTGreleaser** [comment]
Results for commit d94f590ddcb61bf607909656b28cbc10095289a5: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       12 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 5bc0f93a83b2681f6d070a1a2dec769e383deeca: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       12 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 3c266ec25719640bfa0b2054cc1be20ae112f6b3: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       12 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 917666a3e67879f244c381e026526a91058c1b9f: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       12 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit acf64c36c304689530c943034ef1d8145a1274dd: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       12 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 4aa22b4e3fade5430f068c18d0c8c496d21d82a1: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       12 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit bfbff488d7dec8789f8857ea3176fc8ed1bb1e62: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       12 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 4494d45277fe929d5922ad17a0469ef2041ccdf5: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       12 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit d17464287883f19f8bbad5c8704a0eb21d837045: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       12 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**